apps/tests/integration/integration_test.go

```diff
@@ -409,3 +410,37 @@ func TestRemoveAccount(t *testing.T) {
 	}
 
 }
+
+func TestAuthnScheme(t *testing.T) {
```

Comment:
Consider deleting this test--it doesn't add any interesting coverage because it just explicitly specifies the default scheme. Many other tests implicitly test the default scheme and we don't need a live test to validate that custom schemes are used correctly

Reply:
fair enough I removed it.

Reply:
Should I also remove the sample then under `test/devapps` ?

Reply:
If customers shouldn't use this feature, I think we shouldn't provide sample code for it.

Reply:
fair enough. I'll remove it.